src/main/resources/application.properties

```diff
@@ -1,3 +1,3 @@
-azure.documentdb.uri=put-your-documentdb-uri-here
-azure.documentdb.key=put-your-documentdb-key-here
-azure.documentdb.database=put-your-documentdb-databasename-here
+azure.documentdb.uri=@env.DOCUMENTDB_URI@

```

Comment:
modify this will need update root readme.md, is there better way to indicate properties from environment for deploy to jenkins, such as put code snippet of application.properties in deploymentxxx.md?